README.md

```diff
@@ -444,10 +444,29 @@ The following methods allow you to import and export from/to a given XFDF file.
     // Exports all annotations from the current document to the specified XFDF file path in '"~/Documents/" + xfdfPath'.
     exportXFDF(xfdfPath, [callback]);
     
+
+Document Processing API
+-----------------------
+
+The following method allows you to process annotations (embed, remove, flatten, or print) and save the processed document to the given document path.
+
+    // Processes the current document's annotations and saves the processed document at the specified path.
+    processAnnotations(annotationChange, processedDocumentPath, [callback], [annotationType])
+	
+
+The annotationChange is a string parameter. Can be flatten, remove, embed or print.
```

Comment:
Could we add `text` here is well?
```suggestion
The `annotationChange` is a string parameter. Can be `flatten`, `remove`, `embed` or `print`.
```

Or is it against the style?

Reply:
We don't wrap parameter names between backticks in the README. So I left it this way for consistency.

My personal preference is wrap them backticks. I'll push the changes on this PR for the whole README.